assistant/v1.ts

```diff
@@ -113,7 +112,7 @@ class AssistantV1 extends BaseService {
         }
       })
     };
-    return createRequest(parameters, _callback);
+    return this.createRequest(parameters, _callback);
```

Comment:
Is `this` still required, even though `createRequest` is now defined by the superclass instead of an import?